deepchem/models/tensorgraph/layers.py

```diff
@@ -3182,9 +3180,12 @@ def set_tensors(self, tensor):
     self.out_tensor, self.betas = tensor
 
 
-class PassThroughLayer(Layer):
+class LayerSplitter(Layer):
   """
   Layer which takes a tensor from in_tensor[0].out_tensors at an index
+  This is a special layer which only makes sense in the context of TensorGraph.
```

Comment:
Isn't it much more limited than that?  Most layers don't have an `out_tensors` field, so it's only useful with a few specific layers.